app/components/post_attachment_opengraph/post_attachment_opengraph.js

```diff
@@ -169,38 +169,72 @@ export default class PostAttachmentOpenGraph extends PureComponent {
         previewImageAtIndex(this.props.navigator, [this.refs.item], 0, files);
     };
 
-    render() {
-        const {isReplyPost, openGraphData, theme} = this.props;
-        const {hasImage, height, imageUrl, width} = this.state;
+    renderDescription = () => {
+        const {openGraphData} = this.props;
+        if (!openGraphData || !openGraphData.description) {
```

Comment:
I think we don't need to even check for `!openGraphData` as it exits early without that in render

Reply:
That's right and I've observed that too.  However, I thought it would be better to have it that way.  0/5 whether to change it or not, unless you're higher than that.

Reply:
Yes we don’t need to check for opengraphData here

Reply:
Cool, now updated removing null check into it.